packages/runtime/datastore/src/channelStorageService.ts

```diff
@@ -48,6 +49,17 @@ export class ChannelStorageService implements IChannelStorageService {
         return blob ?? this.storage.read(id);
     }
 
+    public async readString(path: string): Promise<string> {
```

Comment:
This implementation looks correct, but in essence it's duplicated code. I'd really prefer for read() to be removed from this  (and other interfaces) in favor of readBlob() and readString to be implemented as add-on either using directly readBlob() (and doing decoding), or through on-the-fly addition of readString to any object as I demoed in my PR. Later seems more versatile, as it allows storage to implement readString if it wants to.